inventory/group_vars/all.yml

```diff
@@ -1,10 +1,14 @@
- # Directory where the binaries will be installed
+# Directory where the binaries will be installed
 bin_dir: /usr/local/bin
 
 # Where the binaries will be downloaded.
 # Note: ensure that you've enough disk space (about 1G)
 local_release_dir: "/tmp/releases"
 
+# Uncomment this line for CoreOS only.
+# Directory where python binary is installed
+# ansible_python_interpreter: "/opt/bin/python"
+
```

Comment:
I don't think that's possible.
Indeed, the first task is to gather facts and ansible needs to know where the python binary is.